test/node/object_id_tests.js

```diff
@@ -24,6 +24,192 @@ describe('ObjectId', function () {
     done();
   });
 
+  it('should correctly create ObjectId from ObjectId', function () {
+    var tmp = new ObjectId();
```

Comment:
this is super nitpicky, but since we're already using `const` in this file anyway, could you update the tests to use `const` and `let` instead of `var`? (`var` is the legacy style javascript)